### PR TITLE
PP 68 fix date time in lessons

### PR DESCRIPTION
**Pull request by vrudnitskaya**
### Description
I fixed the timer that wasn't working after adding the date formatting function. I also corrected the date formatting function itself.
### Screenshots
timer
lessons-teacher

### CODE REVIEW

**Review by mariyador — APPROVED** [review]
Thank you, Valya, for fixing that!

**Review by ofeterovskaya — APPROVED** [review]
Thank you Valya for catching this bug and fixing it

**Review by AnnaPestova1 — COMMENTED** [review]
Hi Valya!
Thank you for returning the functionality. 
Please, could you fix the time issue. Ny lesson is at 9 am tomorrow, it is 10.45 am right now. I see this notice:
Your lesson starts in
(0d 25 hours).

File: src/util/LessonsTimer.jsx [hunk 1]
```diff
@@ -16,9 +16,9 @@ const CountdownTimer = ({ targetDate }) => {
     const now = new Date();
     const difference = targetDate - now;
     const days = Math.floor(difference / (1000 * 60 * 60 * 24));
-    const hours = Math.floor(
+    const hours = (Math.floor(
       (difference % (1000 * 60 * 60 * 24)) / (1000 * 60 * 60)
-    );
+    ))+16;
```
Comment (AnnaPestova1) [inline src/util/LessonsTimer.jsx:21]: I think adding hardcoded 16 can be an issue

**Review by AnnaPestova1 — CHANGES_REQUESTED** [review]
Hi Valya! 
Could you please check the comments and fix the bug.
Thank you

File: src/util/LessonsTimer.jsx [hunk 2]
```diff
@@ -14,7 +14,16 @@ const CountdownTimer = ({ targetDate }) => {
 
   function calculateTimeRemaining(targetDate) {
     const now = new Date();
-    const difference = targetDate - now;
+    const localTargetDate = new Date(
+      targetDate.toLocaleString('en-US', { timeZone: 'Asia/Tokyo' })
```
Comment (AnnaPestova1) [inline src/util/LessonsTimer.jsx:18]: cool! seems you find location)

**Review by AnnaPestova1 — APPROVED** [review]
Valya! You did it! Congrats

**Pull request merged**